packages/router-plugin/tests/code-splitter.test.ts

```
@@ -234,3 +244,813 @@ export const Route = createFileRoute('/')({
+
+describe('small-scope exhaustive: computeSharedBindings', () => {
+  // Helper to generate a route file with specific binding configurations
+  function makeRouteCode(opts: {
+    bindings: Array<{ name: string; kind: 'const' | 'function' | 'class' }>
+    loaderRefs: Array<string>
+    componentRefs: Array<string>
+    errorComponentRefs?: Array<string>
+    beforeLoadRefs?: Array<string>
+  }): string {
+    const bindingDecls = opts.bindings
+      .map((b) => {
```

Biome (2.3.14)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)